### PR TITLE
Bug 1918438: adds i18n for serving/eventing

### PR DESCRIPTION
**Pull request by invincibleJai**
**Fixes**: 
https://bugzilla.redhat.com/show_bug.cgi?id=1918438

**Analysis / Root cause**: 
i18n translation were not handled a few places

**Solution Description**: 
- Added i18n for default details page
- Added i18n for create yaml
- Added i18n misses for serverless

**Screen shots / Gifs for design review**: 

image

image

image

image

image

image

image

**Browser conformance**: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: frontend/public/components/default-resource.tsx [hunk 1]
```diff
@@ -88,50 +99,58 @@ const TableRowForKind = ({ obj, index, key, style, customData }) => {
...
+};
+
+export const DetailsForKind = (kind: string, t: TFunction): React.FC<DetailsForKindProps> => ({
+  obj,
+}) => {
+  return (
+    <>
+      <div className="co-m-pane__body">
+        <SectionHeading
+          text={t('public~{{kindReference}} Details', { kindReference: kindForReference(kind) })}
```
Comment (vikram-raj) [inline frontend/public/components/default-resource.tsx:113]:
```suggestion
          text={t('public~{{kindReference}} details', { kindReference: kindForReference(kind) })}
```

File: frontend/public/locales/en/public.json [hunk 2]
```diff
@@ -121,6 +122,10 @@
   "Utilization": "Utilization",
   "Network transfer": "Network transfer",
   "Pod count": "Pod count",
+  "None": "None",
+  "{{kindReference}} Details": "{{kindReference}} Details",
```
Comment (vikram-raj) [inline frontend/public/locales/en/public.json:126]: Need to update this json file as well.

**Review by vikram-raj — COMMENTED** [review]
Verified the changes. Translation for all the missing strings has been added.

/lgtm

**Comment by spadgett** [comment]
/override ci/prow/ceph-storage-plugin

The ceph-storage-plugin job should only have run on PRs with changes to `frontend/packages/ceph-storage-plugin`. It failed due to an unrelated CI config change. Overriding status on this PR.

**Comment by karthikjeeyar** [comment]
Verified locally, works fine
/lgtm

**Pull request merged**